openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -780,11 +779,11 @@ The Credential Issuer indicates support for encrypted responses by including the
 
 ### Proof Types {#proof-types}
 
-This specification defines the following values for the `proof_type` property:
+This specification defines the following proof types:
 
-* `jwt`: A JWT [@!RFC7519] is used as proof of possession. When `proof_type` is `jwt`, a `proof` object MUST include a `jwt` claim containing a JWT defined in (#jwt-proof-type).
-* `cwt`: A CWT [@!RFC8392] is used as proof of possession. When `proof_type` is `cwt`, a `proof` object MUST include a `cwt` claim containing a CWT defined in (#cwt-proof-type).
-* `ldp_vp`: A W3C Verifiable Presentation object signed using the Data Integrity Proof as defined in [@VC_DATA_2.0] or [@VC_DATA], and where the proof of possession MUST be done in accordance with [@VC_Data_Integrity]. When `proof_type` is set to `ldp_vp`, the `proof` object MUST include a `ldp_vp` claim containing a [W3C Verifiable Presentation](https://www.w3.org/TR/vc-data-model-2.0/#presentations-0) defined in (#ldp-vp-proof-type).
+* `jwt`: A JWT [@!RFC7519] is used for proof of possession. This implies for `proof` objects that the `proof_type` parameter is set to `jwt` and a `jwt` parameter contains a JWT defined in (#jwt-proof-type) MUST be used. This implies for `proofs` objects that a `jwt` parameter containing an array of JWTs defined in (#jwt-proof-type) MUST be used.
+* `cwt`: A CWT [@!RFC8392] is used for proof of possession. This implies for `proof` objects that the `proof_type` parameter is set to `cwt` and a `cwt` parameter contains a CWT defined in (#cwt-proof-type) MUST be used. This implies for `proofs` objects that a `cwt` parameter containing an array of CWTs defined in (#cwt-proof-type) MUST be used.
+* `ldp_vp`: A W3C Verifiable Presentation object signed using the Data Integrity Proof [@VC_Data_Integrity] as defined in [@VC_DATA_2.0] or [@VC_DATA] is used for proof of possession. This implies for `proof` objects that the `proof_type` parameter is set to `ldp_vp` and a `ldp_vp` parameter containing a [W3C Verifiable Presentation](https://www.w3.org/TR/vc-data-model-2.0/#presentations-0) defined in (#ldp-vp-proof-type) MUST be used. This implies for `proofs` objects that an `ldp_vp` parameter containing an array of [W3C Verifiable Presentation](https://www.w3.org/TR/vc-data-model-2.0/#presentations-0) defined in (#ldp-vp-proof-type) MUST be used.
```

Comment:
```suggestion
* `jwt`: A JWT [@!RFC7519] is used for proof of possession. When a `proof_type` parameter in a `proof` object is set to `jwt`, it MUST also contain a `jwt` parameter that includes a JWT as defined in (#jwt-proof-type). When a `proofs` object is using a `jwt` proof type, it MUST include a `jwt` parameter containing an array of JWTs as defined in (#jwt-proof-type).
* `cwt`: A CWT [@!RFC8392] is used for proof of possession. When a `proof_type` parameter in a `proof` object is set to `cwt`, it MUST also contain a `cwt` parameter that includes a CWT as defined in (#cwt-proof-type). When a `proofs` object is using a `cwt` proof type, it MUST include a `cwt` parameter containing an array of CWTs as defined in (#cwt-proof-type).
* `ldp_vp`: A W3C Verifiable Presentation object signed using the Data Integrity Proof [@VC_Data_Integrity] as defined in [@VC_DATA_2.0] or [@VC_DATA] is used for proof of possession. When a `proof_type` parameter in a `proof` object is set to `ldp_vp`, it MUST also contain an `ldp_vp` parameter that includes a [W3C Verifiable Presentation](https://www.w3.org/TR/vc-data-model-2.0/#presentations-0) defined in (#ldp-vp-proof-type). When a `proofs` object is using a `ldp_vp` proof type, it MUST include an `ldp_vp` parameter containing a [W3C Verifiable Presentation](https://www.w3.org/TR/vc-data-model-2.0/#presentations-0) defined in (#ldp-vp-proof-type).
```

there might be even cleaner way to say it but at least trying to replace `it implies..`